lib/channel.rb

```diff
@@ -0,0 +1,74 @@
+require 'httparty'
+require 'awesome_print'
+require 'dotenv'
+Dotenv.load
+
```

Comment:
Your `Channel` class here is just a collection of class methods.  It would be better if you had a class to represent a channel with instance variables etc. The `get_channels` method could then return an array of channel instances.  That would also allow you to have an instance method to send messages, or get details.